CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# On Github Issues and Pull Requests
+
+Found a bug? Want to contribute changes to the codebase? Make sure to read this first.
```

Comment:
That's kind of TBD. At this point, very little new functionality is being added to Keras core. So, if it is new functionality, then it can probably be put here.

There will still need to be some case-by-case weighing on the maintainers' part to determine if something is worth adding. Step 1 of the Pull Requests section reminds people to open an issue first, to see if it is worth implementing.